tests/ts/liblastlog2/write_read_user

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="write_read_user"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+
+TESTPROG="$TS_HELPER_LIBLASTLOG2_WRITE_READ_USER"
+
+[ -x $TESTPROG ] || ts_skip "test not compiled"
+
+ts_run $TESTPROG >> $TS_OUTPUT.debug 2>&1
+
+rm "$TS_TOPDIR"/../tst-write-read-user.db
```

Comment:
No need for the full path to the db file.
The helpers dump them into the current directory, so they can be referenced with it.